tests/phpunit/CiviTest/CiviUnitTestCase.php

```diff
@@ -3722,4 +3722,61 @@ public function enableMultilingual() {
     $dbLocale = '_en_US';
   }
 
+  /**
+   * Build and return pcpBlock params.
+   *
+   * Create the necessary initial objects for a pcpBlock, then return the
+   * params needed to create the pcpBlock.
+   *
+   */
+  public function pcpBlockParams() {
```

Comment:
I'm increasingly using Traits in the test classes to share functions like this that you want in more than one class but not in EVERY class

Reply:
Oh, very nice. I've never used traits before. I just make that switch.